tests/unit/test_stages.py

```diff
@@ -162,6 +247,221 @@ def test_validation_passes(self):
         messages = validation.messages
         self.assertEqual(len(messages), 0)
 
+    @ddt.data(
+        ([10], ["q1"], [], ReviewState.NOT_STARTED),
+        ([10], ["q1"], [mri(USER_ID, "q1", peer=10, answer='1')], ReviewState.COMPLETED),
+        ([10], ["q1"], [mri(OTHER_USER_ID, "q1", peer=10, answer='1')], ReviewState.NOT_STARTED),
+        (
+                [10], ["q1", "q2"],
+                [mri(USER_ID, "q1", peer=10, answer='1'), mri(OTHER_USER_ID, "q1", peer=10, answer='1')],
+                ReviewState.INCOMPLETE
+        ),
+        (
+                [10], ["q1"],
+                [mri(USER_ID, "q1", peer=10, answer='2'), mri(USER_ID, "q2", peer=10, answer="1")],
+                ReviewState.COMPLETED
+        ),
+        (
+                [10], ["q1", "q2"],
+                [mri(USER_ID, "q1", peer=10, answer='3')],
+                ReviewState.INCOMPLETE
+        ),
+        (
+                [10], ["q1"],
+                [mri(USER_ID, "q2", peer=10, answer='4'), mri(USER_ID, "q1", peer=11, answer='5')],
+                ReviewState.NOT_STARTED
+        ),
+        (
+                [10, 11], ["q1"],
+                [mri(USER_ID, "q1", peer=10, answer='6'), mri(USER_ID, "q1", peer=11, answer='7')],
+                ReviewState.COMPLETED
+        ),
+        (
+                [10, 11], ["q1", "q2"],
+                [mri(USER_ID, "q1", peer=10, answer='7'), mri(USER_ID, "q1", peer=11, answer='8')],
+                ReviewState.INCOMPLETE
+        ),
+    )
+    @ddt.unpack
+    def test_review_status(self, peers_to_review, questions, reviews, expected_result):
+        self.project_api_mock.get_peer_review_items_for_group.return_value = reviews
+
+        with mock.patch.object(self.block_to_test, 'review_subjects', mock.PropertyMock()) as patched_review_subjects, \
+                mock.patch.object(self.block_to_test, 'required_questions', mock.PropertyMock()) as patched_questions:
+            patched_review_subjects.return_value = [ReducedUserDetails(id=rev_id) for rev_id in peers_to_review]
+            patched_questions.return_value = [make_question(q_id, 'irrelevant') for q_id in questions]
+
+            self.assertEqual(self.block.review_status(), expected_result)
+            self.project_api_mock.get_peer_review_items_for_group.assert_called_once_with(
+                self.workgroup_data.id, self.activity_mock.content_id
+            )
+
+    def _set_project_api_responses(self, workgroups, review_items):
+        def workgroups_side_effect(user_id, course_id):  # pylint:disable=unused-argument
```

Comment:
You probably already know this, but depending on pylint configuration you can also just prefix unused argument names with an `_` to suppress the `unused-argument` warning.